account/views.py

```diff
@@ -0,0 +1,26 @@
+from django.shortcuts import redirect, render
+
+
+from .forms import UserRegisterationForm
+# Create your views here.
+
+def home(request):
+    return render(request, 'account/default.html')
+
+
+def register_user(request):
```

Comment:
I think it is better if we use class based views

Reply:
updated